pkg/helpers/pointers.go

```diff
@@ -0,0 +1,9 @@
+package helpers
+
+func IsStringPtrNilOrEmtpy(ptr *string) bool {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function IsStringPtrNilOrEmtpy should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)